content/en/faqitems/fleet-management-2-scanning-cluster-software-vulns.markdown

```diff
@@ -0,0 +1,7 @@
++++
+question = "How do I scan my cluster for software vulnerabilities?"
+group = "Fleet Management"
++++
+
+Host vulnerability scanning capability is included with Bottlerocket by using the {{< ver-ref project="os" page="/install/quickstart/aws/host-containers/index.markdown" >}}control container{{< /ver-ref >}} to utilize [Amazon Inspector](https://docs.aws.amazon.com/inspector/latest/user/scanning-ec2.html).
```

Comment:
Is this a better control container link for this context?

https://github.com/bottlerocket-os/bottlerocket#control-container

Reply:
sentence structure needs work.

'using' ... 'utilize' (different forms of the same word really close together).